_layouts/post.html

```diff
@@ -1,58 +1,66 @@
 ---
 layout: default
+slug: guides
 custom_js:
-- prism
+  - prism
+  - guides
 ---
 
-<div>
-  <div class="container-fluid section-ghostwhite">
-    <div class="col-md-3">
-      <div id="all-guides">
-        <label class="pull-right" >
-          <a href="/guides">&lt; All Guides</a>
-        </label>
+<div  class="container-fluid post-bg">
+  <div class="post-detail">
+    <div class="col-sm-2-5">
+      <div id="toc">
+          {% toc %} {{page.document | tocify_asciidoc}}
       </div>
-        <div id="toc">
-            {% toc %} {{page.document | tocify_asciidoc}}
-        </div>
-    </div>
-    <div class="col-md-7 post-section-white">
-        <div class="col-md-9 col-md-offset-2 post-title text-center">
-            <img src="{{ page.image }}" alt="post logo">
-            <label>{{ page.title }}</label>
-            <p>
-                {% assign words = content | number_of_words %}
-                {% if words < 360 %}
-                  1 min read.
-                {% else %}
-                  {{ words | divided_by:180 }} mins read.
-                {% endif %} 
-                Updated {{ page.date | date_to_string }}
-            </p>
-            {% if page.tags %}
-              {% for tag in page.tags %}
-                <span class="badge badge-pill badge-default">
-                  {% capture tag-name %}{{tag | upcase }}{% endcapture %}
-                  {% if tag-name=="AWS" or tag-name=="GCP"  or tag-name=="GKE" %} {{ tag-name }} {% else %} {{tag-name | capitalize}}{% endif %}</span>
-              {% endfor %}
-            {% endif %}
-          </div>
-          <div class="content">
-              {{ content }}
-          </div>
-
-    </div>
-    <div class="col-md-2">
-        <div class="card contact-us-card text-center">
-          <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span>
-            <div class="card-body">
-              <p class="card-title">Want to learn more? Feel free to contact us for support</p>
-              <a href="#" class="btn btn-primary">Support</a>
-              <a href="#">Dismiss</a>
-            </div>
-          </div>
     </div>
+    <div class="col-md-7 guides-section-white">
+      <ol class="breadcrumb text-center">
+        {% assign crumbs = page.url | remove:'/index.html' | split: '/' %}
+        <a href="/guides" ga-event-category="guide-{{ post.title | downcase}}" ga-event-action="deployment-guides">Dev Ops Guides</a> 
+        {% for crumb in crumbs offset: 2 %}
+          {% if forloop.last %}
+           <a href="#"> / {{ crumb | replace:'-',' ' | remove:'.html' | capitalize }}</a>
+          {% else %}
+            <a href="{{ crumb }}">/ {{ crumb | replace:'-',' ' | remove:'.html' | capitalize }}</a>
+          {% endif %}
+        {% endfor %}      
+      </ol>
+      <div class="post-title text-center">
+        <img class="img-center" src="{{ page.image }}" alt="post logo">
+        <label>{{ page.title }}</label>
```

Comment:
Page title should be an `h1`. Using proper HTML tags is important for accessibility and SEO.